### PR TITLE
Add new Meeting Policy setting - ExternalMeetingJoin enum

### PR DESCRIPTION
**Pull request by kpyapali**
-> Added a new meeting policy setting - ExternalMeetingJoin 

Name: ExternalMeetingJoin
Determines whether the user is allowed to join external meeting.
Possible values are:
- EnabledForAnyone
- EnabledForTrustedOrgs
- Disabled

Default value: EnabledForAnyone

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -801,12 +801,12 @@ Accept wildcard characters: False
 ```
 
 ### -AllowUserToJoinExternalMeeting
-Currently, this parameter has no effect.
+Determines whether the user is allowed to join external meeting.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:804]:
```suggestion
Determines whether the user is allowed to join external meetings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kpyapali, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by kpyapali** [comment]
@yogkumgit - Who can help with merging this PR? 
Seems I don't have write permissions.

**Review by get-itips — APPROVED** [review]

**Comment by get-itips** [comment]
> @yogkumgit - Who can help with merging this PR? Seems I don't have write permissions.

@tiburd

**Comment by tiburd** [comment]
> > @yogkumgit - Who can help with merging this PR? Seems I don't have write permissions.
> 
> @tiburd

I'm working on it. I'll ping you on Teams.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**